Python/initconfig.c

```diff
@@ -63,8 +63,12 @@ static const char usage_3[] = "\
 -W arg : warning control; arg is action:message:category:module:lineno\n\
          also PYTHONWARNINGS=arg\n\
 -x     : skip first line of source, allowing use of non-Unix forms of #!cmd\n\
--X opt : set implementation-specific option. The following options are available:\n\
-\n\
+-X opt : set implementation-specific option (see details with -X help)\n\
+--check-hash-based-pycs always|default|never:\n\
+         control how Python invalidates hash-based .pyc files\n\
+";
+static const char usage_xoptions[] = "\
```

Comment:
For now this string is between usage_3 and usage_4, to minimize the diff.
I think I will move it, and change the formatting as well (don’t need all these leading spaces, may need some blank lines)